### ISSUE TITLE
Add proper italic style.

### ISSUE THREAD

**Issue by hiimsergey** [catppuccin/obsidian#46]
Hello,

At first I would like to thank for the effort put into this gorgeous theme as well as the excellent font choice.

However, I found that italic text looks kind off odd. Comparing with the official font website, [Vollkorn](http://vollkorn-typeface.com), the current implementation lacks its style.

**Italic text on the website:**
![20230807_15h26m41s_grim](https://github.com/catppuccin/obsidian/assets/91432388/1b579f33-a776-446a-a9f0-fbe61a860dea)

**Italic text in my vault:**
![20230807_15h36m26s_grim](https://github.com/catppuccin/obsidian/assets/91432388/dc51cb86-b8c1-4b9c-836e-2e08d7f2f44f)

Basically, it inherits the same serif style from regular text and slants it.

I'm not sure if it's intentional or not and if it's possible to change it.

However, thank you in advance.

**Comment by mbeckrich** [comment]
Awesome, I’ll plan on adding that and crediting you with the fix when I get to the computer in a bit. 😎 On Aug 7, 2023, at 8:35 AM, serg ***@***.***> wrote:﻿
(Accidentally closed the issue, oops.)
I found this base64 encoder at https://amio.github.io/embedded-google-fonts and encoded Vollkorn Italic 400 from Google Fonts and just pasted the output and the end of theme.css. It solved the issue for me.

**Comment by mbeckrich** [comment]
This should be updated as of https://github.com/catppuccin/obsidian/commit/7e853eeb5b4d7022e5497080e47a3489d9ee4b5a. Let me know if you run into further issues. Thanks again for catching this and providing the quick solution.
